packages/addons-website/scss/components/_website-switcher.scss

```diff
@@ -0,0 +1,61 @@
+@import '../prefix';
+
+.#{$prefix}--website-switcher {
+  position: fixed;
+  right: 0;
+  width: 0;
+  background-color: $ibm-colors__gray--100;
+  height: 100%;
+  z-index: 1000;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2);
+  will-change: width;
+  transition: width 0.11s cubic-bezier(0.2, 0, 1, 0.9);
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.#{$prefix}--website-switcher--expanded {
+  width: mini-units(32);
```

Comment:
Are we still using mini-units? 🤔  I don't think its used anywhere else on the site, can we use rem(px) instead.